sources/packages/backend/apps/api/src/app.aest.module.ts

```diff
@@ -77,6 +77,7 @@ import {
   ReportService,
   DisbursementOverawardService,
   ConfirmationOfEnrollmentService,
+  NoteSharedService,
```

Comment:
Why is a shared service created for Notes? Is the note creation going to happen anywhere other than API?

Reply:
Because it is used by the API and in "DisbursementOverawardService" that is in `libs/services`.

Reply:
Makes sense. Thanks for explaining.